packages/web-components/src/horizontalbarchart/horizontalbarchart.ts

```diff
@@ -45,14 +44,38 @@ export class HorizontalBarChart extends FASTElement {
   @attr({ converter: jsonConverter })
   public data!: ChartProps[];
 
-  @attr
+  @observable
   public uniqueLegends: ChartDataPoint[] = [];
 
   @attr
   public hideRatio = false;
 
+  @attr({ attribute: 'hide-legends', mode: 'boolean' })
+  public hideLegends?: boolean;
```

Comment:
add a default value for the attribute

Reply:
added in https://github.com/microsoft/fluentui/pull/33199